administration/src/bp-modules/stores/StoresButtonBar.tsx

```diff
@@ -0,0 +1,43 @@
+import { Button, Tooltip } from '@blueprintjs/core'
+import React, { ReactElement } from 'react'
+
+import ButtonBar from '../ButtonBar'
+import { AcceptingStoreEntry } from './AcceptingStoreEntry'
+
+type UploadStoresButtonBarProps = {
+  goBack: () => void
+  acceptingStores: AcceptingStoreEntry[]
+  importStores: () => void
+}
+
+const getToolTipMessage = (hasAcceptanceStores: boolean, hasValidStores: boolean): string => {
+  if (!hasAcceptanceStores) {
+    return 'Laden sie bitte eine Datei mit Akzeptanzpartnern hoch.'
+  }
+  if (!hasValidStores) {
```

Comment:
Nit-picky naming suggestion:
`if (hasInvalidStores)`
Because the current condition reads "if there is no valid stores"

Reply:
adjusted it